### PR TITLE
this navbar is alright we need to change logo and color

### PR DESCRIPTION
**Pull request by diegolarraz**
@Annedj

### CODE REVIEW

File: app/views/shared/_navbar.html.erb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+          <i class="fas fa-leaf"></i> Reuseful
+        <% end %>
+      </div>
+      <nav>
+        <ul class="main-menu">
+          <li><a href="#section-2">About</a></li>
+          <li><a href="#section-7">Story</a></li>
+          <li><a href="#section-9">FAQs</a></li>
+          <li><a href="#main-footer">Contact</a></li>
+          <% if @user %>
```
Comment (Annedj) [inline app/views/shared/_navbar.html.erb]: Shouldn't we use if current_user here instead?

Comment (diegolarraz) [reply app/views/shared/_navbar.html.erb]: yea, I was waiting for you guys to tell me what the variable name is

Comment (diegolarraz) [reply app/views/shared/_navbar.html.erb]: because we have no back end

Comment (diegolarraz) [reply app/views/shared/_navbar.html.erb]: Anne

File: app/assets/stylesheets/components/_navbar.scss [hunk 2]
```diff
@@ -10,3 +16,47 @@
...
+      text-align: center;
+      li {
+        display: inline-block;
+        padding: 0.75em 1.875em;
+        a {
+          padding-bottom: 0.75em;
+          text-transform: uppercase;
+          color: $main-color;
+          font-size: 0.875em;
+          &:hover {
```
Comment (oneillre) [inline app/assets/stylesheets/components/_navbar.scss]: Please change to one level of nesting for ease of readability

File: app/views/shared/_navbar.html.erb (hunk 1 above)
Comment (Annedj) [reply app/views/shared/_navbar.html.erb]: that'll be fine if we add @user = current_user in the Pagescontroller#home method

**Pull request merged**